scripts/enabled_ftr_configs.js

```diff
@@ -10,11 +10,24 @@ require('../src/setup_node_env');
 
 var yaml = require('js-yaml');
 var fs = require('fs');
+var path = require('path');
+
+var manifestsJsonPath = path.resolve(__dirname, '../.buildkite/ftr_configs_manifests.json');
+console.log(manifestsJsonPath);
```

Comment:
Should we use an actual logger, instead of console?

Reply:
Ahh, it's probably not loaded yet right?

Reply:
yes, it is a vanilla js script that you added a while ago, my change broke it :)

Reply:
I added?  I can't recall, but I believe ya

Reply:
Oh I see now.